docs/metrics/v3.md

```diff
@@ -20,378 +22,384 @@ Some metrics are bucket specific. These will have a `/bucket` component in their
 
 Instead of a metrics scrape, it is also possible to list the metrics that would be returned by a path. This is done by adding a `?list` query parameter. The MinIO server will then list all possible metrics that could be returned. During an actual metrics scrape, only available metrics are returned - not all of them. With the `list` query parameter, the output format can be selected - just set the request `Content-Type` to `application/json` for JSON output, or `text/plain` for a simple markdown formatted table. The latter is the default.
```

Comment:
The parameter is listed as `?list` in one spot and then `list` in the other.
To avoid confusion, we should list it the same way in both places.

Reply:
Removed the ? with the expectation that anybody reading this knows what a query parameter is. An example is a good idea too.